opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -77,6 +82,30 @@ export const objects = async (context: AuthContext, user: AuthUser, workspaceId:
   return listThings(context, user, types, finalArgs);
 };
 
+export const exportObjectsAsStixReportBundle = async (context: AuthContext, user: AuthUser, workspace: BasicStoreEntityWorkspace): Promise<string> => {
+  if (workspace.type !== 'investigation') {
+    throw FunctionalError('You can only export investigation objects as a stix report bundle');
+  }
+  const types = ['Stix-Core-Object', 'Stix-Core-Relationship', 'Stix-Sighting-Relationship'];
+  const filters = [{ key: buildRefRelationKey(RELATION_HAS_REFERENCE), values: [workspace.id] }];
+  const stixObjects = await stixLoadByFilters(context, user, types, { filters });
+  const reportData = {
+    name: workspace.name,
```

Comment:
Can overlap an existing report at import (maybe is a KO limit)

Reply:
This was requested in the brainstorm to name the report after the investigation